docs/chapter_heap/heap.md

```diff
@@ -606,7 +638,7 @@ comments: true
         // 判空处理
         if h.isEmpty() {
             fmt.Println("error")
-			return nil
```

Comment:
same as above

Reply:
I'm very sorry about this, here it's the vscode plugin that automatically typesets the code and I didn't notice the error